### PR TITLE
OSDOCS#8887: 4.14.5 MicroShift z-stream

### PR DESCRIPTION
**Pull request by kelbrown20**
**Version(s):** 4.14

**Issue:** [OSDOCS-8887](https://issues.redhat.com//browse/OSDOCS-8887)

**Link to docs preview:**

[MicroShift 4.14 release notes -> RHBA-2023:7601 - MicroShift 4.14.5 bug fix update advisory](https://68864--docspreview.netlify.app/microshift/latest/microshift_release_notes/microshift-4-14-release-notes#microshift-4-14-5-dp)

**QE review:**
- No QE required

### CODE REVIEW

File: microshift_release_notes/microshift-4-14-release-notes.adoc [hunk 1]
```diff
@@ -361,4 +361,18 @@ For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers
...
+Issued: 2023-12-05
+
+{product-title} release 4.14.5, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:7601[RHBA-2023:7601] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:7599[RHSA-2023:7599] advisory.
+
+For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel9/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel9].
+
+[id="microshift-4-14-5-bug-fixes"]
+==== Bug fixes
+
+* Previously, when MicroShift was installed on a non-ostree system, an error was logged during greenboot health check stating that the ostree directory did not exist. MicroShift still started normally. Now, the system properly logs the ostree status as “not an ostree system” and an error message is not produced.
```
Comment (skopacz1) [inline microshift_release_notes/microshift-4-14-release-notes.adoc:378]: I'm guessing based on line 367 that these MicroShift mentions should be attributes:
```suggestion
* Previously, when {microshift-short} was installed on a non-ostree system, an error was logged during greenboot health check stating that the ostree directory did not exist. {microshift-short} still started normally. Now, the system properly logs the ostree status as “not an ostree system” and an error message is not produced.
```

**Review by skopacz1 — COMMENTED** [review]
Just one nit but otherwise LGTM!

File: microshift_release_notes/microshift-4-14-release-notes.adoc (hunk 1 above)
Comment (kelbrown20) [reply microshift_release_notes/microshift-4-14-release-notes.adoc:378]: Whoops, great catch!

**Pull request merged**